src/styles/main.scss

```diff
@@ -1,3 +1,50 @@
-body {
-  margin: 0;
+@import "./variables.scss";
+@import "./reset";
+@import "./normalise";
+
+.month {
+  padding: 10px;
+  width: 7*$cell-size + $cell-size/10*2 + 6px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $free-spase;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $primdry-color;
+    font-size: $num-of-day-size;
+
+    @for $date from 1 through 31 {
+      &:nth-child(#{$date}) {
+        &::before {
+          content: "#{$date}";
+        }
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
+      transform: translateY(-20%);
+      transition-duration: 500ms;
```

Comment:
Why cursor: pointer; only for :hover pseudoclass?

Also, checklist:
9. [STYLES] - Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/6290d4bc-98e8-4889-890e-0cb7535972d4)